### PR TITLE
docs(README): remove how to use CD on TEST/PROD hosting

### PR DESCRIPTION
**Pull request by sbancal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,46 +16,6 @@ The following organization permission should be set:
...
-
-jobs:
-  deploy:
-    runs-on: [self-hosted, deploy, test]
-    steps:
-      - uses: EPFL-ENAC/epfl-enac-deploy-action@main
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-```
+## Known instances
```
Comment (blueur) [inline README.md:19]: instances ?!

**Review by blueur — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sbancal) [reply README.md:19]: a better word for this ?

Comment (blueur) [reply README.md:19]: Continuous Deployment Documentation ?

Comment (sbancal) [reply README.md:19]: To me, the purpose here is not to link to the documentation.
`github-actions-runner` is a generic project that allows us to implement such runners. I would have found it useful to list here all known instances of it. You see what I mean ?

Regarding the documentation, our internal docs will directly point to the right place, so we don't need to add a link here in my opinion.

**Comment by sbancal** [comment]
This is outdated now.

**Pull request closed without merging**